### DISCUSSION TITLE
Add Author field to Product elements

### DISCUSSION THREAD

**Discussion by sjcallender** [craftcms/commerce#2738] [Ideas]
Within PR #1905, @lukeholder [said](https://github.com/craftcms/commerce/pull/1905#issuecomment-872391489) an Author field would not be added to Products, because

> Products are not owned by an author like Entries.

I think this is incorrect thinking about the role of the Author field within Entries. I've never understood the Author field to identify the Owner of an entry. Instead, it's been helpful to identify who created the entry.

In the same way, knowing who created a Product element would be extremely valuable for a multi-person team adding content. It helps to know who did what.

**Comment by mdunbavan** [comment]
To further the support for this feature. I have a real world example that I think would be beneficial to so many ecom sites running on Craft Commerce.

I have a client running a shop that has multiple brands on it, these brands would love to be added to the shop to contribute on articles and manage product inventory from time to time. This feature would be hugely beneficial.

The other area that I think is partly overlooked in this is the power this would add to multi-tenancy ecom sites and marketplaces for products sold by larger umbrella organisations. This area is growing in ecom not just for physical product but for digital products like courses for example. 

Before you completely ignore this idea and feature I would delve a little deeper into how much power this could add to Commerce as a whole. A small feature with mighty benefits to the ecosystem IMO.

**Comment by elivz** [comment]
Passing along another request from a client for this. As products get closer and closer to Entries in terms of functionality (versioning, multi-site, etc) it would be very useful to be able to track who created a product for historical & auditing purposes.
